CHAGELOG-template.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+_These releases can contain markup changes and potentially breaking changes.
+A :warning: icon indicates a change that might be a breaking change for your project._
+
+## [Version #]
+<!-- 
+Every release should have an entry.
+Releases should be listed in reverse chronological order, with the newest release listed first.
+-->
+
+[Release date]
+<!-- 
+Include the date the version was released.
+The date should be in the following format: January 1, 2000. -->
+
+[Itemized list of all user-affected changes]
+<!--
+Example format:
+```
```

Comment:
Might not need the back ticks since it's a comment.

Reply:
Backticks removed in https://github.com/uswds/uswds-team/pull/207/commits/9bd41cdf4efbfb2d3fe3c3606b5712e87fccf49e